### ISSUE TITLE
Feature: Debug test

### ISSUE THREAD

**Issue by jdelrue** [formulahendry/vscode-dotnet-test-explorer#16]
Right click -> Debug test

Runs the test with debug possibilities (stops at breakpoints)

**Comment by arpit-nagar** [comment]
Strongly needed functionality.
Do you have any plans to implement this?

**Comment by stefanforsberg** [comment]
Not at the moment I'm afraid

**Comment by plsgard** [comment]
Maybe it can help : https://medium.com/@mikezrimsek/debugging-dotnet-core-projects-with-visual-studio-code-ff0ab66ecc70

**Comment by stefanforsberg** [comment]
@plsgard Yes, we actually have something in the works that uses something similar to that post =)

**Comment by BukeMan** [comment]
@stefanforsberg Is this still under development?

**Comment by semenmiroshnichenko** [comment]
This is already implemented, isn't it? 
![image](https://user-images.githubusercontent.com/3511513/93081530-aa87bc80-f68f-11ea-95fb-660121c388ea.png)

**Comment by nicolastakashi** [comment]
Any possibility to have an option to debug all tests?

Great work :D
